CMakeLists.txt

```diff
@@ -0,0 +1,126 @@
+#----------------------------------------------------------------------------------------
```

Comment:
There's a lot of stuff in this PR not relevant to embeddings. Please remove the `LLama.GenerateSolution` folder.

Reply:
> There's a lot of stuff in this PR not relevant to embeddings. Please remove the `LLama.GenerateSolution` folder.

Would you be able to make a separate PR for https://github.com/SciSharp/LLamaSharp/pull/674/commits/9c91fac20f3ebde5d1f1bc6a9feacaaa61c4d087 please? Maybe you can do it as admin.
This is GitHub merging all of my PRs. I do not like it very much.

Reply:
I can't do that, it's coming from your fork which I have no admin rights over.

Reply:
I could split it. It is removed from here.